src/cljs/netrunner/gameboard.cljs

```diff
@@ -1183,7 +1183,9 @@
                       [:button {:on-click #(send-command "choice" {:choice c})}
                        (for [item (get-message-parts c)] (create-span item))]
                       (let [[title code] (extract-card-info (add-image-codes (:title c)))]
-                        [:button {:on-click #(send-command "choice" {:card @c}) :id code} title]))))))]
+                        (if (:rotated c)
+                          [:button.rotated {:on-click #(send-command "choice" {:card @c}) :id code} title]
+                          [:button {:on-click #(send-command "choice" {:card @c}) :id code} title])))))))]
```

Comment:
You could probably rewrite this to use `{:class (when (:rotated c) :rotated) ...}` instead of the `if` statement, which might look cleaner.

Reply:
This is nicer.  I hate FAQ time.